plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/exporter/SpanExporterProvider.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing.exporter;
+
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import org.opensearch.common.settings.Settings;
+
+/**
+ * Helps in creating the SpanExporter.
+ */
+public interface SpanExporterProvider {
```

Comment:
@Gaganjuneja we do not need this addition abstraction, otherwise every single new exporter requires code changes on the plugin side:
 - we should be able to use `SpanExporter` directly (if the implements it)
 - the convention for installatiation could be: factory methods `create`, `getInstance`, no-arg constructor

The settings seems to be not used at the moment by the exporters but span processors, so we could probably leave this aspect aside.

Reply:
Pushed the changes.